astropy/io/ascii/core.py

```diff
@@ -652,10 +652,10 @@ def check_column_names(self, names, strict_names, guessing):
             # Impose strict requirements on column names (normally used in guessing)
             bads = [" ", ",", "|", "\t", "'", '"']
             for name in self.colnames:
-                if (_is_number(name) or
-                    len(name) == 0 or
-                    name[0] in bads or
-                    name[-1] in bads):
+                if ((_is_number(name) or
+                     len(name) == 0 or
+                     name[0] in bads or
+                     name[-1] in bads)):
```

Comment:
While PEP8 doesn't explicitly recommend against 4-space indentation after an `if-` statement, most linters (e.g. `flake8`) will at least raise a warning for it.